src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -2015,6 +2010,7 @@ private void SetupMicrobePartSelections()
             var control = (MicrobePartSelection)organelleSelectionButtonScene.Instance();
             control.Locked = organelle.Unimplemented;
             control.PartIcon = organelle.LoadedIcon ?? throw new Exception("Organelle with no icon");
+            control.CanBeModified = organelle.CanBeModified;
```

Comment:
Ah yeah, now I remember that this was separate from the tooltip. So this is also needed.
I guess we decided that for now showing the modifiable icon on the organelles that can be modified is fine. It'll be easy enough to make this an option / disable it once almost every single organelle can be modified.